src/misc/DefaultAvatar/DefaultAvatar.js

```diff
@@ -28,13 +28,18 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-export const DefaultAvatar = ({ userName, size }) => {
+export const DefaultAvatar = (props) => {
+  const { userName, size, variant } = props;
```

Comment:
Same here we can use destructuring directly mike this :
export const DefaultAvatar = ( { userName, size, variant }) => {